source/tag-developers/textarea-tag.md

```diff
@@ -1,5 +1,5 @@
 ---
-layout: default
+layout: tag-developers
 title: Tag Developers Guide
 ---
 
```

Comment:
The rest of the textarea tag content was _not_ changed in this PR, but when viewed in the staged site it looks _different_ to the other tags in the _tag-reference_ section.  The other tags seemed to have been modified for the description/attribute but not the textarea.

Maybe changing:

`__Description__`  to  `## Description`
`__Parameters__`   to  `## Attributes`

would make textarea consistent with the other tags in _tag-reference_ ?

Reply:
Looks like I had started but didn't finish this page transformation.